float32[3] dpos
float32 yaw